src/payment/strategies/bluesnapv2/bluesnapv2-payment-page-loader.ts

```diff
@@ -0,0 +1,59 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+
+import { NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+
+import { BlueSnapV2PaymentInitializeOptions } from './bluesnapv2-payment-options';
+
+export default class BlueSnapV2PaymentPageLoader {
+    private _blueSnapV2Options?: BlueSnapV2PaymentInitializeOptions;
+    private _formPoster = createFormPoster();
+
+    initialize(options?: BlueSnapV2PaymentInitializeOptions): void {
```

Comment:
I don't think you need an initialize method for `BlueSnapV2PaymentPageLoader ` ?

It seems that you could so just call `loadPaymentPage(options?: BlueSnapV2PaymentInitializeOptions)` and include frameName inside `BlueSnapV2PaymentInitializeOptions`